src/ManageCourses.Domain/DatabaseAccess/ManageCoursesDbContext.cs

```diff
@@ -243,12 +244,63 @@ public void AddMcUser(McUser user)
         {
             McUsers.Add(user);
         }
+        public List<UcasCourse> GetUcasCourseRecordsByUcasCode(string instCode, string ucasCode, string email)
+        {
+            var ucasCourses = UcasCourses.FromSql(
+                    $"select c.* from ucas_course c " +
```

Comment:
Don't use `*`. It's fragile and hard to troubleshoot.
I don't think using raw sql is advisable unless there's an exceptionally good reason for it. We lose all of the strong type checking and ability to find usages etc that we gain from sticking to c# & linq.

Reply:
:+1: